source/StellarObject.h

```diff
@@ -72,10 +73,17 @@ class StellarObject : public Body {
 	// Find out how far this object is from its parent.
 	double Distance() const;
 
+	// Change the distances an object is visible from, based on flagship attributes.
+	void UpdateDistanceVisibility(const Ship *flagship);
+
 
 private:
 	const Planet *planet;
 
+	// The base values that are used to generate the visibility distances in Body.
+	double trueDistanceVisible = 0.;
+	double trueDistanceInvisible = -1.;
+
```

Comment:
```suggestion
```
As these can only be set to the body's visibility ranges, I don't think they are necessary anymore.